lib/multi_factor_auth.rb

```diff
@@ -8,6 +8,8 @@ class Disabled < MFAError; end
   class NotConfigured < MFAError; end
 
   def self.valid?(phone_number)
+    return false unless phone_number
```

Comment:
I think https://github.com/alphagov/govuk-account-manager-prototype/blob/main/app/controllers/devise_registration_controller.rb#L73 is also slightly wrong, as it checks the parameter is not nil, but then passes `nil` to this method if the parameter is the empty string.

Reply:
Having validation here is good but we should also change the check in the controller to check for the presence of the number correctly.

Reply:
Aha, the `presence` is changing this from an empty string to `nil` in the validation.  I'll change the first part to `.present?`.

Reply:
I've decided to just remove the first part of the `if` here.  It seems to have served no purpose since `.presence` handles that anyway.

Reply:
I'm not sure the `.presence` is adding anything anymore, given that the method now checks for `nil` and the number validation will (I hope) reject the empty string; but it's not doing any harm.